plugin/converter/starlark/starlark.go

```diff
@@ -16,11 +16,9 @@ package starlark
 
 import (
 	"bytes"
-	"context"
-	"errors"
-	"strings"
 
 	"github.com/drone/drone/core"
+	"github.com/drone/drone/handler/api/errors"
 	"github.com/sirupsen/logrus"
 	"go.starlark.net/starlark"
```

Comment:
package grouping (github.com/drone should be separate from third party packages)